### PR TITLE
Added mummy_finance optimism trades

### PR DESCRIPTION
**Pull request by ARDev097**
Added trades data for mummy_finance optimism dex

### CODE REVIEW

**Comment by Hosuke** [comment]
Hi @ARDev097, do you want to use the macro approach to implement `mummy_finance_optimism.base_trades` as well?

You can find some sample PRs in #4759

**Comment by ARDev097** [comment]
Hello @Hosuke ,
I've encountered an error in the 'dex_trades_assert_freshness.sql' file after implementing the macro approach in the Mummy Finance DEX. I've reviewed my changes, but I'm unable to pinpoint the issue. Could you please take a look and provide guidance on how to correct it?

image

**Comment by Hosuke** [comment]
The [data freshness issue](https://dune.com/queries/3233799) is unrelated to this PR.

File: tests/dex/dex_trades_assert_freshness.sql
Comment (Hosuke) [inline tests/dex/dex_trades_assert_freshness.sql:1]: Maybe we can recover this file to avoid data freshness check in this PR?

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @ARDev097 

[Sample mummy_finance_optimism_base_trades]()

**Comment by Hosuke** [comment]
> Hello @Hosuke , This PR still hasn't been merged. Please review it, and inform me if there is anything that needs to be done on my end.

Just noticed all tests passed now, I will pass this PR to final review.

**Review by jeff-dude — APPROVED** [review]
thank you for the patience 🙏

**Pull request merged**